infrastructure/modules/ses/main.tf

```diff
@@ -0,0 +1,42 @@
+resource "aws_ses_domain_identity" "ndr_ses" {
+  domain = "${terraform.workspace}.${var.domain}"
+  count  = var.enable ? 1 : 0
+}
+
+resource "aws_ses_domain_dkim" "ndr_dkin" {
+  domain = aws_ses_domain_identity.ndr_ses[0].domain
+  count  = var.enable ? 1 : 0
+
+}
+
+resource "aws_route53_record" "ndr_amazonses_dkim_record" {
```

Comment:
could we rename anything `amazonses` to just `ses` please?

Reply:
Do you mean in the record itself? or just the name in terraform?
  records = ["${aws_ses_domain_dkim.ndr_dkin[0].dkim_tokens[count.index]}.dkim.amazonses.com"]
I think this needs to be amazonses